src/i18n/l10n/en.ftl

```diff
@@ -494,6 +494,12 @@ Identifiers-View = Identifiers View
 Identify-an-organism = Identify an organism
 Identify-an-organism-with-the-iNaturalist-AI-Camera = Identify an organism with the iNaturalist AI Camera
 If-an-account-with-that-email-exists = If an account with that email exists, we've sent password reset instructions to your email.
+# Explanation that observations are removed from a collection project
+If-you-leave-x-of-your-observations-removed =
+    If you leave this traditional project, { $count ->
+        [one] 1 of your observations
+       *[other] { $count } of your observations
+    }  will also be removed from this project.
```

Comment:
You have an extra space before "will" here.

Reply:
Thanks. Then it does matter how many spaces around the parens are? With one before and one after it will show one or two in the text?

Reply:
Space outside the curly brace will render. I noticed it when testing this text in the bottom sheet.